comment/comment_repository_blackbox_test.go

```diff
@@ -51,6 +52,35 @@ func (s *TestCommentRepository) createComments(comments []*comment.Comment, crea
 	}
 }
 
+func (s *TestCommentRepository) TestCreateCommentWithParentComment() {
+	// parent comment
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.Identities(1))
+	parentComment := newComment(uuid.NewV4(), "Test A", rendering.SystemMarkupMarkdown)
+	s.repo.Create(s.Ctx, parentComment, fxt.Identities[0].ID)
+	// child comments
+	childComment := newComment(uuid.NewV4(), "Test Child A", rendering.SystemMarkupMarkdown)
+	childComment.ParentCommentID = id.NullUUID{
+		UUID:  parentComment.ID,
+		Valid: true,
+	}
+	// when
+	s.repo.Create(s.Ctx, childComment, fxt.Identities[0].ID)
+	// then
+	assert.NotNil(s.T(), childComment.ID, "Comment was not created, ID nil")
```

Comment:
Instead of `assert`, `require` should be used because `require` is used to enforce a precondition. If the precondition fails, the test should fail and should not move ahead for further checking.

Reply:
Fixed.